gnomad/utils/vcf.py

```diff
@@ -865,3 +890,64 @@ def set_female_y_metrics_to_na(
             }
         )
     return female_metrics_dict
+
+
+def build_vcf_export_reference(
+    name: str,
+    build: str = "GRCh38",
+    contigs_keep: List[str] = [f"chr{i}" for i in range(1, 23)]
+    + ["chrX", "chrY", "chrM"],
+) -> hl.ReferenceGenome:
+    """
+    Create export reference based on reference genome defined by `build`.
+
+    By default this will return a new reference with all non-standard contigs eliminated. Keeps chr 1-22, Y, X, and M.
+
+    Non-standard for example is: ##contig=<ID=chr3_GL000221v1_random,length=155397,assembly=GRCh38>
+
+    :param name: Name to use for new reference.
+    :param build: Reference genome build to use as starting reference genome.
+    :param contigs_keep: Contigs to keep from reference genome defined by `build`.
+    :return: Reference genome for VCF export containing only contigs in `contigs_keep`
```

Comment:
```suggestion
    :return: Reference genome for VCF export containing only contigs in `contigs_keep`.
```
(adding period for consistency with other docstring lines)